hm_pyhelper/miner_json_rpc/client.py

```diff
@@ -1,7 +1,14 @@
 import requests
+import base64
+import base58
+
+import hm_pyhelper.protos.blockchain_txn_pb2 as blockchain_txn_pb2
+import hm_pyhelper.protos.blockchain_txn_add_gateway_v1_pb2 \
+        as blockchain_txn_add_gateway_v1_pb2
 from hm_pyhelper.miner_json_rpc.exceptions import MinerConnectionError
 from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedURL
 from hm_pyhelper.miner_json_rpc.exceptions import MinerRegionUnset
+from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedAddGatewayTxn
```

Comment:
I'm sure this is intentional for readability, would it be better if we condensed the 3 lines using `hm_pyhelper.miner_json_rpc.exceptions`

Not really a functional change but more so with styling